helm-chart/renku/values.yaml

```diff
@@ -178,11 +178,15 @@ ingress:
   ## Enables the creation of an ingress
   enabled: false
 
+  className: nginx
+
+  certManager:
```

Comment:
This doesn't work. The annotations are for the ingress, they are not necessarily related to `certManager`.
To achieve what you want here you can simply remove the two annotations: `cert-manager.io/cluster-issuer` and `kubernetes.io/ingress.class:`

Reply:
It does work. This new field has dedicated handling in the `ingress` template which will add the additional annotations.

The original idea was to clearly separate the handling of cert-manager dedicated annotations from the defaults provided by the chart itself.

Reply:
What I mean is what should one do if they want to add annotation to the ingress regardless of cert-manager?
Also why are you proposing this change?
What problem are you trying to solve?

Reply:
While writing this answer, I found out the issue that I had in mind but that I did not address properly (thanks for making me think a bit further): if one wants to have additional annotations, they currently have to copy the content from the `values.yaml` file and then add their own annotations.

```yaml
ingress:
  enabled: true
  annotations:
    # original values
    nginx.ingress.kubernetes.io/proxy-body-size: "0"
    nginx.ingress.kubernetes.io/proxy-request-buffering: "off"
    nginx.ingress.kubernetes.io/proxy-buffer-size: "8k"
   # user custom values
```
If all these annotations shall be mandatory for Renku to operate properly (at least the last two seems to be), then I would rather expect the chart to give the option of only changing their values (for example, `ingress.proxyBufferSize: "8k"`) so that the ingress always has the minimal set of annotations required. On top of that, you can give the option for additional annotations which could use something like `extraAnnotations`. These `extraAnnotations` would then be a good fit for the `cert-manager` annotations.

I am proposing this change to help administrators and developers have an easier time to deploy and test renku and also to avoid breaking it unintentionally.